Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1464,6 +1464,30 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
 
                 // Update the wheel's visibility
                 m_pVehicle->SetWheelVisibility((eWheelPosition)ucWheel, (ucStatus != DT_WHEEL_MISSING));
+
+                // restart wheels component visibility (fix #1556)
+                static const SString strWheelPrefix = "wheel_";
+
+                for (const auto& [key, value] : m_ComponentData)
+                {
+                    SString strComponentName = key;
```

Comment:
`strComponentName` is just a copy of `key`. This copy is unnecessarily, you can use `key` directly. I also recommend to change `key` and `value` to more reasonable names.

Reply:
I found a much simpler way to fix this bug 😄